Dockerfile

```diff
@@ -9,4 +9,4 @@ COPY public ./public
 COPY src ./src
 RUN npm run build
 EXPOSE 1337
-CMD ["npm", "run", "start"]
+CMD ["npm", "run", "develop"]
```

Comment:
This is also used to build the staging image in https://github.com/dev-launchers/strapiv4/blob/main/.github/workflows/staging-image.yaml#L47. Can we override the entrypoint in https://github.com/dev-launchers/strapiv4/blob/main/docker-run.sh#L2 instead

Reply:
I added a separate dev dockerfile and used it to build the dev docker image in the makefile